frontend/src/pages/CourseSelector/CourseTabs/styles.js

```diff
@@ -2,14 +2,25 @@ import styled from "styled-components";
 
 const CourseTabsWrapper = styled.div`
   background-color: ${({ theme }) => theme.purpleLight};
-  padding-left: calc(20vw - 2px); // 20vw - width of menu sidebar, 2px is tab margin-left
+  padding-left: calc(2vw - 2px); // 20vw - width of menu sidebar, 2px is tab margin-left
   padding-right: 25px;
   width: 100vw;
   height: var(--cs-tabs-cont-height);
   display: flex;
   align-items: center;
 `;
 
+const ShowAllCourses = styled.div`
+  display: flex;
```

Comment:
I think a better approach is for the `ShowAllCourses` container to use `position: absolute` within the `CourseTabsWrapper` where it has `position: relative`. And `ShowAllCourses` could have a `left: 20px` attribute to position it. This is because for higher res screens, it is not positioned exactly to the left:
<img width="364" alt="Screen Shot 2022-07-21 at 10 38 59 am" src="https://user-images.githubusercontent.com/65492346/180105671-b427ec70-cf88-4e04-b3ee-bccf06f3a025.png">

Think having absolute position would give us more exact precision :)